### PR TITLE
Windows doc

### PR DESCRIPTION
**Pull request by andipavllo**

### CODE REVIEW

File: src/documents_en/guide/overview.html [hunk 1]
```diff
@@ -2090,6 +2090,22 @@ <h4 id="BuildingOnsenUI">Building Onsen UI</h4>
 
 If you need to build Onsen UI manually, there is a `gulp` task for that purpose. Please follow the instruction from the [project Website](https://github.com/OnsenUI/OnsenUI).
 
+<h3 id="WindowsDevelopmentKnownIssues">Windows Development Known Issues</h3>
+
+Since version 1.3, Onsen UI supports Windows platform apps development.
+
+Here is a list of known issues, which we recommend to read before planning your app development:
+
+* **Rendering:**  the rendering of the animations (transactions) might be laggy on low performance devices.
```
Comment (argelius) [inline src/documents_en/guide/overview.html]: transition

File: src/documents_en/guide/overview.html [hunk 2]
```diff
@@ -2090,6 +2090,22 @@ <h4 id="BuildingOnsenUI">Building Onsen UI</h4>
...
+
+* **Glitches:** some elements might be affected by graphical glitches when rendered.
```
Comment (argelius) [inline src/documents_en/guide/overview.html]: This is not very helpful. Please write which components and what kind of glitches you've experienced. Also, I would like an GitHub issue for these glitches so we can solve them.

File: src/documents_en/guide/overview.html [hunk 3]
```diff
@@ -2090,6 +2090,22 @@ <h4 id="BuildingOnsenUI">Building Onsen UI</h4>
...
+
+* **Pull Hook:** works, but it's affected by poor scrolling performance when associated with a `ons-list` element.
```
Comment (argelius) [inline src/documents_en/guide/overview.html]: Have you tried solving this with the `touch-action` rule?

**Comment by andipavllo** [comment]
I think I'll try to make another full test based on Onsen UI 1.3.2 and post the results, probably something changed

**Comment by masahirotanaka** [comment]
Not many issues specific to Windows anymore. Aborting to merge :)

**Pull request closed without merging**